### PR TITLE
ESET VM extension

### PR DESCRIPTION
**Pull request by abokov**
ESET VM extensions json file and simple samples

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@abokov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: eset-vm-extension/eset-template.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "vmName": {
+            "type": "string",
+            "defaultValue": "myVM",
+            "metadata": {
+                "Description": "Vitual machine's unique name."
```
Comment (singhkay) [inline eset-vm-extension/eset-template.json]: Virtual :)

File: eset-vm-extension/eset-template.json [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            ],
+            "metadata": {
+                "description": "This is the location where the availability set will be deployed"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D1",
+            "metadata": {
+                "Description": "Virtual machine size"
```
Comment (singhkay) [inline eset-vm-extension/eset-template.json]: Lower case "description"

**Comment by singhkay** [comment]
Looks good generally, only few issues to address

**Pull request merged**